igb_client/parse_custom.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict, List, Set
+from collections import defaultdict
+
+
+def params_parser(payload: Dict) -> Dict[str, List[Any]]:
```

Comment:
is the return signature correct?

Reply:
Indeed, initially it was a list, but it was redundant (due to single child) so changed it into an object